app/assets/src/components/Login.jsx

```diff
@@ -102,92 +102,108 @@ class Login extends React.Component {
   }
 
   sendMail() {
-    var link =
-      "mailto:regger@chanzuckerberg.com?Subject=Request%20IDseq%20Access";
+    var link = "mailto:accounts@idseq.net?Subject=Request%20IDseq%20Access";
```

Comment:
Can we send the email from the server? 
If not, maybe obfuscate a bit with `<a href='mailt&#111;&#58;&#109;a&#105;lt%6F%3A&#97;ccou%6E&#116;s&#64;i&#100;&#115;eq&#46;net&#63;&#83;ub&#106;e%6&#51;%&#55;4=&#82;&#101;qu&#37;&#54;5st%&#50;5&#50;0ID&#37;73e%&#55;1%2&#37;3&#48;Acces&#115;'>&#109;&#97;&#105;&#108;to&#58;accoun&#116;s&#64;idseq&#46;net?Subject=Req&#117;es&#116;%2&#48;ID&#115;eq%20&#65;c&#99;&#101;&#115;s</a>`